libs/components/lookup/src/assets/locales/resources_en_US.json

```diff
@@ -31,14 +31,26 @@
     "_description": "Label for lookup's show more modal add button",
     "message": "New"
   },
+  "skyux_lookup_show_more_select_all_button_aria_label": {
```

Comment:
@blackbaud-johnly - I recognize that it is a little odd that the titles here are `lookup` when this is for the selection modal. However, these strings are shared by the old lookup modal and are only listed once so that we aren't retranslating for no reason. I stuck with `lookup` here as we will be implementing this new input with lookup in the next PR and I wanted to maintain consistency.